### DISCUSSION TITLE
Week 7 - Project Task - Craig Butikofer

### DISCUSSION THREAD

**Discussion by CraigButikofer** [USU-ITLS-x265-23-summer/class-repo#223] [General]
@kensnyder I have not been able to connect with you about my project ideas, but I'll list what I have and you can let me know what you think: 

Main Project Idea: Website for In-Area Advisors

What this would entail: 

This is sort of a massive project that I just want to begin to tackle with this webpage. I'd love to start by creating just a general information page that could reference a couple of pictures and video and help orient people for the FSY program. This would eventually get to the point where they can access training, but it will start as a landing page. Another thing that I'd like to do at the start is to introduce the team that I work with. This might include just a picture and a short bio. I'd love to also include some survey elements that would allow for us to collect some basic information such as username, area, etc. 

I think that I'm hoping to work mainly in HTML and then do styling in CCS, but I could use some tips on how to appropriately incorporate JavaScript. 

Let me know what you think.

**Comment by derekscholes** [comment]
Java really got me.... I literally only figured out how to do some menu stuff and not much else. You can look at some other projects which have integrated it better than me which show resizing effects which is really cool. Then you can ask them how they added it into their sites. I also highly suggest using flex-box in your css. I didn't understand it until i started playing around with making columns and it was very helpful.

**Comment by CJ3315** [comment]
For this project I think that it is best to just stick with HTML and CSS as that will get you your main functionality.

**Comment by kensnyder** [comment]
> @kensnyder I have not been able to connect with you about my project ideas, but I'll list what I have and you can let me know what you think:

A general information page for the FSY program would work great. The other features you can finish after the semester.

**Comment by audreyhki** [comment]
Creating a general information page with images and videos would be a great start to your project. Good luck!

**Comment by kassyoveson** [comment]
Hi Craig. You did your entire project in 2 weeks! That is very impressive. You must be superman or something. Just kidding.
